Doc/library/typing.rst

```diff
@@ -1976,6 +1976,11 @@ Other concrete types
 
    .. versionadded:: 3.5.2
 
+   .. deprecated:: 3.11
+      Python 2 is no longer supported, and most type checkers also no longer
+      support type checking Python 2 code. Users should now use
+      :class:`str` instead of ``Text`` wherever possible.
```

Comment:
Could we add an explicit note here saying that removal is not currently planned? A "deprecated" note can make users think it will be removed soon.

Reply:
(For future readers of this PR):

- #92405